css/style.css

```diff
@@ -15,5 +14,234 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+header,
+main,
+
+footer {
+    padding: 0 1.5rem;
+    max-width: 1200px;
+    margin: 0 auto;
```

Comment:
In the picture, the image touches the side of the screen, so to achieve that, this should have no padding or max-width